tgfilestream/web_routes.py

```diff
@@ -63,8 +63,21 @@ async def handle_request(req: web.Request, head: bool = False) -> web.Response:
         return web.Response(status=404, text="404: Not Found")
 
     size = message.file.size
-    offset = req.http_range.start or 0
-    limit = req.http_range.stop or size
+    try:
+        offset = req.http_range.start or 0
+        limit = req.http_range.stop or size
+        if (limit > size) or (offset < 0) or (limit < offset):
+            raise ValueError("range not in acceptable format")
+    except ValueError:
```

Comment:
Why is this not just an `if`?

Reply:
If http range is not correct, aiohttp raises value error. Also raising value error if limit is greater than size. Thus we can catch both errors in one try.